src/interfaces/toMarcRecords.js

```diff
@@ -27,7 +28,11 @@ export async function streamToMarcRecords({correlationId, headers, stream}) {
 				// Operation CREATE -> f001 new value
 				if (operation === OPERATIONS.CREATE) {
 					// Field 001 value -> 000000000, 000000001, 000000002....
-					updateField001ToParamId(`${recordNumber}`, record);
+					record = updateField001ToParamId(`${recordNumber}`, record);
+
+					await amqpOperator.sendToQueue({queue: correlationId, correlationId, headers, data: record.toObject()});
+
+					return;
```

Comment:
You can just return the Promise directly in one line (`return` instead of `await`).